### ISSUE TITLE
probelms in comparing code to article "Playing FPS Games with Deep Reinforcement Learning"

### ISSUE THREAD

**Issue by natangold85** [glample/Arnold#9]
as far as i understand according to the article there are supposed to be two neural nets. 
one DQN for navigation scope decision and the other DRQN for action scope decision and feature extraction. in the code i don't see the option of 2 networks loading simultaneously. also in the next_action function i don't see any use of the game_features that extracted from DRQN net.

**Comment by philipjball** [comment]
I believe that Arnold2 (i.e., this repo) didn't have a separate network: https://arxiv.org/pdf/1809.03470.pdf

**Comment by yculcarnee** [comment]
@fiorenza2 Can you point me to the repo with the two networks seperate? I too have the same issue as OP and would like to understand more of the navigation part of their implementation. Thanks im advance :)

**Comment by philipjball** [comment]
Hey, not sure there is a public repo with the original Arnold code with navigation network; it might be best to reach out to the original authors to see if they can help?

**Comment by yculcarnee** [comment]
@fiorenza2 perhaps yeah, I'll try to do that. Thanks! :)

**Comment by fabero** [comment]
@yculcarnee Did you get any response from the authors? We are experiencing the same issue when trying to reproduce the strategy outlined in the paper.

**Comment by glample** [comment]
Hi,

Sorry, we did not include the two networks in this code. We decided to merge the navigation and deathmatch rewards into the same model. This way, it is harder to control how much the model is exploring / camping etc., but this is much easier to train and to evaluate. We also found that merging everything into a single network can also work well, although it requires a much more careful reward shaping.

You can still use this code to train 2 networks for exploration / deatchmatch, but you will have to merge them afterwards.
